### PR TITLE
Update tutorial for REST API proxy

### PR DESCRIPTION
**Pull request by RukshiW**
Resolves https://github.com/wso2-enterprise/choreo/issues/14212

### CODE REVIEW

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md [hunk 1]
```diff
@@ -1,94 +1,121 @@
 # Create Your First REST API Proxy
 
-Choreo is a low-code cloud-native platform for building integration solutions. In this tutorial, you will do the following to explore REST API Proxies in Choreo:
+This tutorial introduces REST API proxies. 
+
+An API proxy fronts an API backend and intercepts the requests to the REST API. You can apply security, rate limiting, etc., to the REST API Proxy.
```
Comment (shaniR) [inline en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:5]: This sentence is a little confusing.. it says it fronts the Bakcend,, and then it says it intercepts the requests to the REST API.  Shall we say, "An API Proxy exposes an existing API on Choreo and intercepts the requests to the REST API."

Comment (shaniR) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:5]: Shall we say, " The API Proxy acts as a managed API to which you can apply security, rate-limiting, etc."

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md [hunk 2]
```diff
@@ -1,94 +1,121 @@
...
+
 ## Step 1: Create a REST API Proxy
 
 You can create a REST API proxy in one of the following methods: 
 
-- Design a new REST API proxy using an existing endpoint. 
-- Importing an OpenAPI specification of the existing API.
 
-In this tutorial, you will be importing an OpenAPI specification and creating a REST API proxy:
+In this tutorial, you will be providing a URL to a sample OpenAPI definition of a sample
```
Comment (shaniR) [inline en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:25]: This part is a little confusing: sample OpenAPI definition of a sample. Can we reword?

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md [hunk 3]
```diff
@@ -1,94 +1,121 @@
...
-    |-----------------|-----------------------|
-    | **Input Type** | OpenAPI URL |
-    | **OpenAPI URL** | `https://petstore3.swagger.io/api/v3/openapi.json` |
+7. Click **Create**. 
+
+The API proxy opens on a separate page.
+
+## Step 2: Developing the REST API Proxy
+
+Let's develop the REST API proxy by adding an operation. To do this, follow the steps given below:
```
Comment (shaniR) [inline en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:50]: Let's design? develop is generally implementing

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md [hunk 4]
```diff
@@ -1,94 +1,121 @@
...
+    5. Click **Save**.
 
-2. Let's deploy the REST API proxy to the development environment. Click **Config & Deploy** in the **Build Area** column.
+3. Click **API Definition**. You will see the new operation you added in the OpenAPI definition.
 
-3. Once you have deployed it to the development environment, you can test it. When you are ready to take it to production, you can come back to this view and promote it by clicking on the **Promote** button on the **Development** column.
+## Step 3: Deploy the REST API Proxy
 
-    ![Promote REST API proxy](../../assets/img/tutorials/promote-rest-api-proxy.png){.cInlineImage-threeQuarter}
+Deploying the REST API proxy makes it invokable. By default, Choreo maintains two environments for development and production. [Learn more](../../develop/components/api-proxies/api-proxy#choreo-environments).
```
Comment (shaniR) [inline en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:75]: Shall we add a sentence to say what we will be doing? For e.g. for the purpose of this tutorial let's deploy on the redevelopment environemnt.

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md [hunk 5]
```diff
@@ -149,27 +177,31 @@ Now your REST API proxy is ready to be consumed. An application developer can di
...
-3. Click **Generate Credentials**. As a result, Choreo populates the **Consumer Key** and **Consumer Secret** fields with the newly generated tokens.
+3. Click **Generate Credentials**. Choreo populates the **Consumer Key** and **Consumer Secret** fields with newly generated tokens.
 
 4. Click **://Curl**. Then copy the cURL command that is displayed.
 
     !!!info
-        Click the button to copy the cURL command. **Do not manually copy it**. The displayed cURL command is only a template, and it does not include the generated credentials.
+        Click the icon for copying to copy the cURL command. **Do not manually copy it**. The displayed cURL command is only a template, and it does not include the generated credentials.
+
+5. Close the **Get cURL to Generate Access Token** card. Then click **Update**.
```
Comment (shaniR) [inline en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:191]: Let's mention that this way is solely for testing purposes. And for prod, we have to use the token endpoint with a suitable grant type.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 5 above)
Comment (RukshiW) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:191]: Done

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 4 above)
Comment (RukshiW) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md]: Done

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 3 above)
Comment (RukshiW) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:50]: But the UI itself uses the word "Develop" for proxies.

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md]: Changed to "In this tutorial, you will be providing a URL to an OpenAPI definition of a sample API. "

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md]: Done. Did a minor change to the 2nd suggestion.

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 3 above)
Comment (shaniR) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:50]: Ack

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md [hunk 6]
```diff
@@ -149,27 +177,34 @@ Now your REST API proxy is ready to be consumed. An application developer can di
...
 
-5. Close the **Get cURL to Generate Access Token** card. Then click **Update**. 
     A message appears to confirm that the keys are successfully updated.
+
+    !!! info
+        Here, you are generating an access token solely for testing purposes. When you deploy the API proxy to the production environment, you need to use the token endpoint with a suitable grant type.
```
Comment (shaniR) [inline en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:196]: Sorry I think I made a mistake here. Here when we invoke the cURL we are already using the token endpoint.

File: en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md (hunk 5 above)
Comment (shaniR) [reply en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md:191]: Let's remove this.

**Review by shaniR — APPROVED** [review]

**Review by shaniR — APPROVED** [review]

**Pull request merged**